vnext/Common/Common.vcxproj

```diff
@@ -25,6 +25,14 @@
       <Configuration>Release</Configuration>
       <Platform>ARM</Platform>
     </ProjectConfiguration>
+    <ProjectConfiguration Include="Debug|ARM64">
```

Comment:
@JunielKatarn Would you be willing to pull these out into a separate prop file? I'm about to add a new project, and this duplication is getting to be a bit much.

Reply:
Sorry. I had this in `automerge` and just got in.